src/types.tsx

```diff
@@ -654,29 +664,18 @@ export interface SearchBarProps {
    * A callback that gets called when the text changes. It receives the current text value of the search bar.
    */
   onChangeText?: (e: NativeSyntheticEvent<TextInputFocusEventData>) => void;
-
-  /**
-   * A callback that gets called when search bar is closed
-   *
-   * @platform android
-   */
-  onClose?: () => void;
-  /**
-   * A callback that gets called when search bar has received focus
-   */
-  onFocus?: (e: NativeSyntheticEvent<TargetedEvent>) => void;
   /**
    * A callback that gets called when search bar is opened
    *
    * @platform android
    */
   onOpen?: () => void;
   /**
-   * A callback that gets called when the search button is pressed. It receives the current text value of the search bar.
+   * A callback that gets called when search bar is closed
+   *
+   * @platform android
    */
-  onSearchButtonPress?: (
-    e: NativeSyntheticEvent<TextInputFocusEventData>
-  ) => void;
+  onClose?: () => void;
```

Comment:
Hey, theese props were sorted alphabetically, I think we should restore that

Reply:
I've sorted them by their priority, but as you think we should revert, we should revert then 👍

Reply:
Resolved in https://github.com/software-mansion/react-native-screens/pull/2154/commits/922dbfd773c6d83b7eaf8a13914308eb80fa403f.